nilearn/connectome/tests/test_connectivity_matrices.py

```diff
@@ -357,26 +510,31 @@ def test_sym_matrix_to_vec():
     n = 5
     p = n * (n + 1) // 2
     rand_gen = np.random.RandomState(0)
+
     # when diagonal is included
     vec = rand_gen.rand(p)
     sym = vec_to_sym_matrix(vec)
+
     assert_array_almost_equal(sym_matrix_to_vec(sym), vec)
 
     # when diagonal given separately
     diagonal = rand_gen.rand(n + 1)
     sym = vec_to_sym_matrix(vec, diagonal=diagonal)
+
     assert_array_almost_equal(
         sym_matrix_to_vec(sym, discard_diagonal=True), vec
     )
 
     # multiple matrices case when diagonal is included
     vecs = np.asarray([vec, 2.0 * vec, 0.5 * vec])
     syms = vec_to_sym_matrix(vecs)
+
```

Comment:
I might be missing something, but I feel like we could split these in multiple tests (or a single test with parametrization maybe...). WDYT ?

Reply:
yes probably.

will slit further